RecoVertex/PrimaryVertexProducer/python/OfflinePrimaryVertices_cfi.py

```diff
@@ -11,11 +11,12 @@
     
     TkFilterParameters = cms.PSet(
         algorithm=cms.string('filter'),
-        maxNormalizedChi2 = cms.double(20.0),
+        maxNormalizedChi2 = cms.double(10.0),
         minPixelLayersWithHits=cms.int32(2),
         minSiliconLayersWithHits = cms.int32(5),
-        maxD0Significance = cms.double(5.0), 
+        maxD0Significance = cms.double(4.0), 
         minPt = cms.double(0.0),
+        maxEta = cms.double(2.4),
```

Comment:
May well be possible. This tuning was done with and for phase 1 and the parameters in this file are chosen for 2017 data at high pile-up. My understanding was, that it can can be overridden for other scenarios.